ignite/templates/app/files/go.mod.plush

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/cometbft/cometbft v0.37.2
 	github.com/cometbft/cometbft-db v0.8.0
 	github.com/cosmos/cosmos-proto v1.0.0-beta.2
-	github.com/cosmos/cosmos-sdk v0.47.3
+	github.com/cosmos/cosmos-sdk v0.47.5
```

Comment:
Guess one-line change here is not enough, need also bump cosmossdk.io packs: math, depinject & errors at least (if not missed something)

Reply:
Hm, can't resolve/close my conversation... but mentioned issues resolved.